pywwt/layers.py

```diff
@@ -0,0 +1,100 @@
+import re
+
+class ImageryLayers():
+
+    def __init__(self, layer_list):
+        self._layers = {}
+        self._spectrum = ['gamma', 'x', 'uv', 'visible', 'ir', 'micro', 'radio', 'other']
+        for band in self._spectrum:
+            self._layers[band] = {}
+        
+        self._list2dict(layer_list)
+
+    def _list2dict(self, og_list):
+        for layer in og_list:
```

Comment:
I think it would be good to include a comment here about what this method does and what ``og_list`` is expected to be.